.github/workflows/ci.yaml

```diff
@@ -120,3 +121,27 @@ jobs:
           BUILDER_NAME: ${{ matrix.builder }}
         run: |
           make ${IMAGE_NAME} BUILDER=${BUILDER_NAME}
+
+  envtest:
+    name: envtest
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - uses: actions/cache@v2
+        with:
+          path: |
+                ~/.cache/go-build
+                ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - name: Install controller-runtime and get envtest binaries
+        run: |
+          go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
+          curl -sSLo envtest-bins.tar.gz "https://go.kubebuilder.io/test-tools/${K8S_VERSION}/$(go env GOOS)/$(go env GOARCH)"
+          tar -zxvf envtest-bins.tar.gz -C $HOME 
+      - name: Run envtest
+        run: KUBEBUILDER_ASSETS=$HOME/kubebuilder/bin/ make envtest
```

Comment:
Maybe just move this in build stage? 

```
        run: KUBEBUILDER_ASSETS=$HOME/kubebuilder/bin/ make envtest
```

could go just next to `make test`?

Remember `Jenkinsfile` too.